project/AppDependencies.scala

```diff
@@ -2,14 +2,14 @@ import sbt._
 
 object AppDependencies {
 
-  val bootstrapVersion = "7.13.0"
+  val bootstrapVersion = "7.19.0"
   val hmrcMongoVersion = "0.74.0"
```

Comment:
This can be upgraded

Reply:
Had a look at this. This would require lots of rework since v1 of hmrc-mongo, the LocalDateTime formatter has been removed in favour of using Instant. Will create a new ticket to address this

Reply:
PRM-2735 created